trending_data/src/main/webapp/unique_window_handler.js

```diff
@@ -76,16 +78,26 @@ export class UniqueWindowHandler {
     this.btnDiv = document.createElement('div');
     this.btnDiv.className = 'toggle-btns';
 
-    /** YouTube button */
     this.ytBtn = document.createElement('button');
     this.ytBtn.textContent = 'YouTube';
-    this.ytBtn.className = 'btn btn-default yt yt-selected';
-    this.btnDiv.appendChild(this.ytBtn);
 
-    /** Twitter button */
     this.twitterBtn = document.createElement('button');
     this.twitterBtn.textContent = 'Twitter';
-    this.twitterBtn.className = 'btn btn-default twitter';
+
```

Comment:
Might be good to add a comment that the assumption here is that at least one of the two has to be supported. It's something we know because otherwise the marker wouldn't show up, but it might not be clear to someone reading the code here.